internal/statestore/backfill.go

```diff
@@ -200,6 +208,48 @@ func (rb *redisBackend) UpdateBackfill(ctx context.Context, backfill *pb.Backfil
 	return nil
 }
 
+// CleanupBackfills removes expired backfills
+func (rb *redisBackend) CleanupBackfills(ctx context.Context) error {
```

Comment:
Should this function mirror the code in frontend_service.go's doDeleteBackfill? If so, I think you are missing the DeindexBackfill call.  We might also want to find a way to write the code once and re-use it.

Reply:
good catch. I will try to reuse that code.